### DISCUSSION TITLE
Is there a way to insert HTML content?

### DISCUSSION THREAD

**Discussion by waifu-master** [hand-dot/labelmake#45] [Q&A]
I've tried using `type: "text"` in the `Template` object with an input of `<strong>test</strong>` and it literally displayed `<strong>test</strong>` instead of a bold **test**. Is there a way to do this, or is there no such feature yet?

**Comment by vuvuzella** [comment]
I dont think there is one yet. There were suggestions on setting text weight but haven't heard if there was a PR for it

**Reply by hand-dot** [reply]
Sorry for the delay in replying.
There is no option to insert HTML.
but you should be able to achieve bold font by setting bold text as a custom font.

https://labelmake.jp/javascript-pdf-generator-library/example#labelmake-example-customfont
